CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing
+
+Thank you for your interest in making it easier to write good CSS3! Here's how
+you can help!
+
+## Open an issue
+
+If you think something's missing, make sure you're not asking for something on
+[this list of bad CSS
+properties](https://gist.github.com/y0ssar1an/bb95223148e486acbe7a#file-
+bad_css). If it's not on that list, open an issue and we'll investigate. we'll
+be monitoring the specs as they're updated on [the W3C
+feed](http://www.w3.org/Style/CSS/current-work.en.html), but I still need your
+help. Let's keep bad code out of the Web!
+
+## Submit a pull request
```

Comment:
Could you shed some light on what your process for getting stuff into master is? Do you cherry pick into master? Should all PRs go to the dev branch? Should they make their own feature branch?